pkgs/build-support/setup-hooks/install-shell-files.sh

```diff
@@ -219,6 +219,12 @@ installShellCompletion() {
         else
             install -Dm644 -T "$arg" "$outPath"
         fi || return
+        # Sanity check file size to avoid assuming completion has succeed but it generate an invalid file.
```

Comment:
Add a parameter/flag to mkDerivation or something like that to ignore this check, or something to change the fileSize threshold 

There are edge cases that the size is intentionally smaller like terraform.